packages/cozy-harvest/src/Manifest.spec.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-env jest */
+import { legacyLoginFields, legacyEncryptedFields, Manifest } from './Manifest'
+
+describe('Manifest', () => {
+  describe('sanitize', () => {
+    it("shouldn't modify if fields is null", () => {
+      const input0 = {
+        fields: null
+      }
+      const result = Manifest.sanitize(input0)
+      expect(result.fields).toBe(null)
+    })
+  })
+
+  describe('sanitizeIdentifier', () => {
+    it('should set role=identifier for login', () => {
+      const input1 = {
+        fields: {
+          login: { type: 'text' },
+          password: { type: 'password' }
+        }
+      }
+      const result = Manifest.sanitizeIdentifier(input1)
```

Comment:
I think we should test only the whole sanitize method, and not the internal functions. By doing so we ensure that the sequence of call will keep the change and that another function will not overwrite some properties.